specification/validated-relationships-service-api.yaml

```diff
@@ -355,11 +355,26 @@ paths:
-          | A single active proxy relationship for two consenting adults with capacity | `performer:identifier`=`9000000010` and `status`=`active` and `_include`=`Consent:performer` | HTTP Status 200 Bundle containing a single active proxy relationship for two consenting adults with capacity |
-          | Multiple active proxy relationships with varying legal basis'              | `performer:identifier`=`9000000017` and `status`=`active` and `_include`=`Consent:performer` | HTTP Status 200 A Bundle containing multiple active proxy relationships with varying legal basis'            |
-          | A single active proxy relationship for a birth mother & child              | `performer:identifier`=`9000000019` and `status`=`active` and `_include`=`Consent:performer` | HTTP Status 200 Bundle containing a single active proxy relationship for a birth mother & child              |
+          | Scenario                                                                   | Request                                                                                                      | Response                                                                                                 |
+          | -------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------| ---------------------------------------------------------------------------------------------------------|
+          | Multiple proxy relationships filtered by 'active' status                   | `performer:identifier`=`9000000017` and `status`=`active`                                                    | HTTP Status 200 Bundle containing a multiple active proxy relationships                                  |
+          | Multiple proxy relationships filtered by 'inactive' status                 | `performer:identifier`=`9000000017` and `status`=`inactive`                                                  | HTTP Status 200 Bundle containing a multiple inactive proxy relationships                                |
+          | Multiple proxy relationships filtered by 'proposed' and 'active' status    | `performer:identifier`=`9000000017` and `status`=`proposed,active`                                           | HTTP Status 200 Bundle containing a multiple proposed and active proxy relationships                     |
+          | Multiple proxy relationships without filtering and details                 | `performer:identifier`=`9000000017`                                                                          | HTTP Status 200 Bundle containing a multiple proxy relationships without details                         |
```

Comment:
I can't get this scenario to return the expected response
`http://127.0.0.1:9000/FHIR/R4/Consent?performer:identifier=9000000017`